### DISCUSSION TITLE
Is the product switcher icon required as part of the new navigation?

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3281] [Q&A] [answered]
Hi Paste team,

I am working with the development team on implementing the new navigation.  Just wondering are we required to have a product switcher menu or is this optional (see screenshot below)?  If required, where do the menu items link to?

![image](https://github.com/twilio-labs/paste/assets/9394413/6aed415c-37f7-4c9d-93ae-2658ca4fa162)

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonge495 

Everything in the Navigation UI kit is a building block to creating any application. All the features are optional if the application doesn't have a need for them. 

Treat it like a big jigsaw puzzle, and pick the pieces you need, and leave out the rest.

**Reply by tonge495** [reply]
Great thank you!

**Reply by tonge495** [reply]
Is there any documentation available yet for the navigation UI kit?

**Reply by SiTaggart** [reply]
Not yet, it's coming very soon. We're just wrapping up some final bits for the UI components before we expose the docs to folks
